lomax/scripts/kill-all.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+while true 
+    do
+        pkill -9 "lomax";
```

Comment:
What's the purpose of this script?  "kill-all.sh" sounds like it should just shut lomax down, but in fact it spins in a loop (consuming 100% CPU most likely) and repeatedly kills it.  If this behavior is intended it would be good to have a comment explaining its purpose—just a sentence or two would suffice.